README.md

```diff
@@ -11,18 +11,38 @@
 
 ## 3<sup>rd</sup> Party SDK Support
 
- * [VRTK.Unity.SteamVR]
- * [VRTK.Unity.OculusUtilities]
+  * [VRTK.Unity.SteamVR]
+  * [VRTK.Unity.OculusUtilities]
 
 ## Introduction
 
-VRTK.Unity.Core is a core collection of design patterns for Unity
-that can be beneficial in VR development.
+VRTK.Unity.Core is a core collection of design patterns for [Unity3d] that can be beneficial in VR development.
```

Comment:
I'm a friend of following what Wikipedia and the likes do: Only the first occurrence of a term should be linked.

This would mean we remove the hyperlink to Unity here and anywhere below as the term is mentioned right at the top already.